src/components/EventPage/RsvpSection.tsx

```diff
@@ -154,6 +176,33 @@ export const SessionsWrapper = ({
             {Intl.DateTimeFormat().resolvedOptions().timeZone}
           </span>
         </div>
+        {sortedSessions.map((session, key) => {
+          const userRsvp = session.rsvps.some(
+            (rsvp) => rsvp.attendeeId === user.id
+          );
+          return (
+            <>
+              {userRsvp && (
+                <details className="group">
+                  <summary className="flex cursor-pointer list-none items-center justify-between font-medium">
+                    See All Rsvps list <TransitioningArrow />
```

Comment:
this should be outside the `map`. is displaying the accordion as many times as the number of rsvps
<img width="430" alt="image" src="https://github.com/anggxyz/convo-app/assets/10832414/9a225a94-6a3b-4c24-9ad6-3e408d971dd3">